scripts/fetch-docs.sh

```diff
@@ -9,7 +9,7 @@ YOCTO_DOCS_LIST=" tasks.rst variables.rst"
 set -e
 cd "$(dirname "$(readlink -f "$0")")/.."
 
-mkdir -p client/resources/docs
+mkdir -p server/resources/docs
```

Comment:
You'll need to edit the other references here from client to server.

Reply:
Fixed.